includes/admin/class-wcmp-export-consignments.php

```diff
@@ -419,13 +419,27 @@ private function setCustomsDeclaration(): void
      */
     private function setPhysicalProperties(): void
     {
-        $extraOptions       = WCX_Order::get_meta($this->order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
-        $digitalStampWeight = $extraOptions['weight'] ?? null;
-        $orderWeight        = $this->order->get_meta(WCMYPA_Admin::META_ORDER_WEIGHT);
+        $extraOptions = WCX_Order::get_meta($this->order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
+        $packageType  = $this->getPackageType();
+
+        if (! $extraOptions) {
+            return;
+        }
+
+        $orderWeight = $this->order->get_meta(WCMYPA_Admin::META_ORDER_WEIGHT);
+        $totalWeight = $this->getTotalWeight($orderWeight);
+
+        if ($packageType === AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP) {
+            $totalWeight = $extraOptions['weight'];
```

Comment:
Misschien is het beter als je de weight uit `$this->orderSettings` laat komen zodat je in de shipment options dezelfde waarde ziet als bij de uiteindelijke export. Als het goed is zit die DPZ weight daar al in maar dat weet ik niet helemaal zeker

Reply:
Done